src/main/java/com/jvnlee/catchdining/domain/restaurant/dto/RestaurantSearchResponseDto.java

```diff
@@ -0,0 +1,31 @@
+package com.jvnlee.catchdining.domain.restaurant.dto;
+
+import com.jvnlee.catchdining.domain.restaurant.model.Address;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class RestaurantSearchResponseDto {
+
+    private Long id;
+
+    private String name;
+
+    private Address address;
+
+    private double rating;
+
+    private int reviewCount;
+
+    public RestaurantSearchResponseDto(RestaurantSearchResultDto r) {
+        this.id = r.getId();
+        this.name = r.getName();
+        this.address = r.getAddress();
+        this.rating = r.getRating();
+        this.reviewCount = r.getReviewCount();
```

Comment:
이미 만들어져 있는 `@AllArgsConstructor` 를 사용하면 좋을 것 같네요!

Reply:
`@AllArgsConstructor` 방식 외에도 `RestaurantSearchResultDto`에서 값을 뽑아서 해당 DTO로 매핑시키는 형태의 생성자가 필요해서 생성했습니다.

Reply:
남긴 리뷰의 뜻은 이런 식으로 바꿔보면 좋을 것 같다는 뜻이었습니다 :)
```suggestion
        this(r.getId(), r.getName(), r.getAddress(), r.getRating(), r.getReviewCount());
```